src/Orckestra.Composer/Services/ComposerContext.cs

```diff
@@ -231,6 +237,10 @@ public virtual string GetEncryptedCustomerId()
             return dto.EncryptedCustomerId;
         }
 
+        // ReSharper disable once InconsistentNaming
+        private readonly bool ValidationPasswordChangedNotDisabled =
```

Comment:
You can call it something like **PasswordChangedValidationEnabled**, this way it is both shorter and ReSharper  won't complain